frontend/.env.example

```
@@ -4,3 +4,4 @@
 
 # バックエンド API の URL（ローカル開発時）
 VITE_SHOKEN_WEBAPI_API_URL=http://127.0.0.1:3001
+VITE_SENTRY_DSN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_SENTRY_DSN key should go before the VITE_SHOKEN_WEBAPI_API_URL key

(UnorderedKey)